### ISSUE TITLE
Druid 0.10.0 GroupBy: quarter Granularity is not supported

### ISSUE THREAD

**Issue by var23rav** [druid-io/druid#4512]
Druid GroupBy query facing issue on new version

**Druid 0.9.2**": Has support for **quarter** _Granularity_, query is working fine there.

But for the same query **Druid 0.10.0**, throwing error 
```
{
	"error": "Unknown exception",
	"errorMessage": "com.fasterxml.jackson.databind.JsonMappingException: Granularity is not supported. [P3M] (through reference chain: io.druid.query.groupby.GroupByQuery[\"granularity\"])",
	"errorClass": "java.lang.RuntimeException",
	"host": null
}
```

**Json Query Params**
```
{
	"queryType": "groupBy",
	"intervals": "2016-11-01T00:00:00+00:00/2017-02-01T00:00:00+00:00",
	"aggregations": [{
		"name": "count",
		"type": "count"
	}],
	"dimensions": [
		"register_date",
		"user_name",
		"name"
	],
	"dataSource": "user",
	"granularity": "quarter",
	"postAggregations": []
}
```

**Comment by gianm** [comment]
This was fixed in #4316 and will be available in 0.10.1. An RC is available now if you want to try it out: http://druid.io/download.html

**Comment by var23rav** [comment]
K, Fine Thanks.
